src/content/url-handling.js

```diff
@@ -54,7 +56,9 @@ export function urlFromState(urlState) {
     case 'calltree':
       query.search = urlState.callTreeSearchString || undefined;
       query.invertCallstack = urlState.invertCallstack ? null : undefined;
-      query.jsOnly = urlState.jsOnly ? null : undefined;
+      query.implementation = urlState.implementation === 'all'
+        ? undefined
```

Comment:
Haven't looked at the docs on the query string generation, but I believe the serialization works like this:

```
{
  foo: null,
  bar: undefined,
  baz: 'bee',
}
```
will produce `?foo&baz=bee`, so I was making it so that the "implementation" will not show in up in the query unless needed.